src/conventions/global_ignoring_tag.md

```diff
@@ -45,3 +45,59 @@ execute as @e[tag=!global.ignore] at @s run function namespace:internal/logic/fu
+>```mcfunction
+> # Examples of usage found
+> # Pragmatic
+> execute as @e[tag=!global.ignore,tag=!global.ignore.kill] run function <...>
+>   ├ kill @s
+>   ├ kill @s
+>   ╰ Kill @s
+>
+> # Use in @s
+> > execute as @e[tag=!global.ignore] run function <...>
+>   ├ kill @s[tag=!global.ignore.kill]
+>   ├ kill @s[tag=!global.ignore.kill]
+>   ╰ Kill @s[tag=!global.ignore.kill]
+> # From this situation it was found that before kill command, there are some commands that must execute with tagged entities global.ignore.kill. But there were no kills in any of the commands at that point.
+>```
+
+> - **Q :** Can I just only have `!global.ignore` in the `kill`, `tp` command? Because if that entity has `[ "faq.maker", "global.ignore", "global.ignore.pos", "global.ignore.kill" ]` it won't be selected anyway. 
+> - **A :** Yes 1 ~~Yes 2~~ and *No* or **shouldn't**. 
+>> - Yes 1 : If it has `!global.ignore` at entry selector This is the same as **# Use in @s**, but you also need to make sure that during execution (man in the middle), there are no changes to the executed `@s` entity like this -> `execute as @e[tag=!global.ignore] run kill @s[tag=!global.ignore.kill]`
+>> - ~~Yes 2~~ you can and *No*: But you need to make sure that none of your entities you forgot to include `global.ignore`. And it can be an issue if some datapacks are certified, there is an entity that only `global.ignore.kill` or `global.ignore.pos` which may be caused by The reviewer overlooked or datapacker accidentally do.
+>> - **Shouldn't :** **Please note that** if a problem occurs which is likely to arise from the case used in `@s` tag for whatever reason we have encountered such as **"Why put another tag too when there are only `!global.ignore` It just filters everything out, so it's useless"**, or **"due to many of tags"**, or **"because you're lazy"**, or whatever, That's not the datapack reviewer's fault. Then you come and tell us that **"We found these cases in your datapack, but we still certify them."** You can't say that either because sometimes we review what we get back. It is the reason that it is useless to the reason that we have given and disrespect for such advice. That means we'll assume you don't care in conventions at all. If it gives you trouble. Why don't you do them yourself? You say some conventions are useless and stupid to use. Why don't you debate those ideas and issues at #convention-ideas? Please reduce your ego and toxic. And reason why Datapack Reviewer often pass some cases? Not because of empathy but considering and analyzing some reasons, some cases are not so serious. But if there is any issue in the future, we cannot ignore it.
```

Comment:
What about this?

```suggestion
> - **A : Will be exempt in cases where **# Use in @s** is true but shouldn't. Because if there is any issue in the future, we cannot ignore it. But don't worry, we'll review and clarify for you.
```
Here I think not complaining would be better.